src/Integration/OracleSample/Program.cs

```diff
@@ -13,11 +13,12 @@ static void Main()
     static Task Start()
     {
         var connection = "Data Source=localhost;User Id=particular; Password=Welcome1; Enlist=false";
-        return EndpointStarter.Start("SqlPersistence.Sample.Oracle", persistence =>
-        {
-            persistence.SqlVariant(SqlVariant.MySql);
-            persistence.TablePrefix("MySql");
-            persistence.ConnectionBuilder(() => new OracleConnection(connection));
-        });
+        return EndpointStarter.Start("SqlPersistence.Sample.Oracle",
+            persistence =>
+            {
+                persistence.SqlVariant(SqlVariant.MySql);
+                persistence.TablePrefix("MySql");
```

Comment:
that was not me https://github.com/Particular/NServiceBus.Persistence.Sql/blob/develop/src/Integration/OracleSample/Program.cs